embedded-graphics/src/pixelcolor/raw/mod.rs

```diff
@@ -65,7 +65,7 @@
 //!
 //! fn main() {
 //!     // Create new image with RGBI colors.
-//!     let image: Image<RGBI> = Image::new(IMAGE_DATA, 2, 2);
+//!     let image: ImageRawLE<RGBI> = ImageRawLE::new(IMAGE_DATA, 2, 2);
```

Comment:
Why did you change this to `ImageRawLE` instead of `ImageRaw`? A `RawU4` image shouldn't be depended on the byte order.

Reply:
Good point. I think this was from a blanket find and replace.